clients/go/gengo.go

```diff
@@ -419,10 +510,12 @@ func parseResponseCode(s *spec.Swagger, op *spec.Operation, capOpID string) stri
 		buf.WriteString(statusCodeDecoder)
 	}
 
-	successType := swagger.SuccessType(s, op)
-	successReturn := "nil, "
-	if successType == nil {
-		successReturn = ""
+	successReturn := ""
+	if successType := swagger.SuccessType(s, op); successType != nil {
```

Comment:
should we factor this out into a helper somewhere? only two cases, but seems like it might be worth it.

i have a PR that tries to clean all this multiple input / output param stuff up, but i haven't done anything with it for a while. we could wait until then or we could try to start cleaning up now